nixos/modules/programs/fish.nix

```diff
@@ -13,6 +13,22 @@ let
       (filterAttrs (k: v: v != null) cfg.shellAliases)
   );
 
+  envShellInit = pkgs.writeText "shellInit" cfge.shellInit;
+
+  envLoginShellInit = pkgs.writeText "loginShellInit" cfge.loginShellInit;
+
+  envInteractiveShellInit = pkgs.writeText "interactiveShellInit" cfge.interactiveShellInit;
+
+  sourceEnv = useTranslated: path:
+  if useTranslated then
```

Comment:
```suggestion
  sourceEnv = path:
  if cfg.useBabelfish then
```

Reply:
Do you prefer internalizing the configuration check? It's true the parameter never takes on another value. I'll make this change.